packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -450,75 +450,115 @@ class _DomainAssociatedLinksPanel extends StatelessWidget {
 }
 
 class _PathCheckTable extends StatelessWidget {
+  const _PathCheckTable({
+    required this.controller,
+  });
+
+  final DeepLinksController controller;
+
   @override
   Widget build(BuildContext context) {
```

Comment:
this build method is rather large. Can you break into helper widgets to make this more readable?